src/main.ts

```diff
@@ -51,17 +53,49 @@ async function run(): Promise<void> {
       }
     }
 
-    try {
-      const octokit = github.getOctokit(token)
-      await octokit.checks.create(createCoverageCheckRequest)
+    const octokit = new Octokit({
+        auth: token,
+    });
```

Comment:
Curious about how `const octokit = github.getOctokit(token)` is different from this.

Reply:
I think using that github object to initialize octokit is a lot cleaner, shouldn't have any different I think.

Reply:
Fixed